src/components/editor/Conditional.js

```diff
@@ -93,13 +93,21 @@ class Age extends Component<Props> {
 
   render() {
     let conditional = ((this.props.conditional: any): AgeConditional);
-    let options = [{id: '==' , text:'==' }, {id: '!=' , text:'!=' }, {id: "<" , text:"<" }, {id: "<=" , text:"<=" }, {id: ">" , text:">" }, {id: ">=", text:">="}];
-
+    let operatorOptions = [{id: '==' , text:'==' }, {id: '!=' , text:'!=' }, {id: "<" , text:"<" }, {id: "<=" , text:"<=" }, {id: ">" , text:">" }, {id: ">=", text:">="}];
+    let unitOptions = [
```

Comment:
This is repeated in about 5 places, consider encapsulating it in a `const` and pulling from there. 

Only reason I didn't do that with the operators was because the operators DO differ between types (this drives me up a wall and I wish they didn't)